draft-ietf-capport-architecture.xml

```diff
@@ -779,11 +764,13 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
     </section>
    </section>
 
-   <section anchor="Acknowledgements" title="Acknowledgements">
+   <section anchor="Acknowledgments" title="Acknowledgments">
+     <t>The authors thank Lorenzo Colitti for providing the majority of the content
```

Comment:
Should this be in a Contribution section? I'm not sure of the etiquette here.

Reply:
Acknowledgments are good.  Contributors is usually used for substantial editorial contributions, not just ideas and feedback.  But both are left to editor discretion.  Add as you see fit.